packages/agent/src/agent/http/index.ts

```diff
@@ -728,7 +746,7 @@ export class HttpAgent implements Agent {
   public async fetchRootKey(): Promise<ArrayBuffer> {
     if (!this._rootKeyFetched) {
       // Hex-encoded version of the replica root key
-      this.rootKey = ((await this.status()) as any).root_key;
+      this.rootKey = ((await this.status()) as JsonObject & { root_key: ArrayBuffer }).root_key;
```

Comment:
What's going on here?  This doesn't seem related to the description of the PR.

Reply:
ah, just cleaning up a linting warning in the file since I was touching it